README.md

```diff
@@ -35,7 +35,7 @@ A Docker image [is built](https://github.com/mybb/mybb-build/blob/master/Dockerf
      1. Create a `secrets.env` file in the main directory with the data repository URL variable:
 
         ```
-        INPUT_FILES_REPOSITORY=git@github.com/mybb/...
+        INPUT_FILES_REPOSITORY=git@github.com:mybb/mybb.git
```

Comment:
We can leave the repository name as `...`, since it would be a private one (where `/input/patches/` are copied from; the main code repository is defined as [`sourceGitRepository`](https://github.com/mybb/mybb-build/blob/2859f6c97c58867f4f25ee8c9fadc3b4a23d6cc5/build.xml#L27)). Looks like we can skip `.git` for readability.
```suggestion
        INPUT_FILES_REPOSITORY=git@github.com:mybb/...
```

Reply:
Since it's just an example I think it is best to have a full one for someone to copy?

> This variable is not needed for public usage or reproduction - we can choose a better name and make it more clear in README at some point.

Right I didn't notice that but following the README its easy to think it is 😃

Reply:
`mybb/additional-input-files.git`? It may be confusing with `mybb/mybb` (the source code is copied directly to the working repository, but doesn't do anything).

Reply:
Yeah that's cool, makes it more clear that people without access to the repo don't need to update the env.